### PR TITLE
Add LEGO City Undercover: The Chase Begins

### PR DESCRIPTION
**Pull request by EzekielRage**
Adds LEGO City Undercover: The Chase Begins.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/lego-city-undercover-the-chase-begins/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "LEGO City Undercover: The Chase Begins"
+description = "LEGO City Undercover: The Chase Begins is an \"open world\" game developed by TT Fusion. While the game is technically open world, it is split into various city districts and needs to load each new district when traveling from one part of the city to another. Released on the Nintendo 3DS in April 2013, the game is based on the LEGO City brand and was published by Nintendo. It serves as a prequel to Lego City Undercover."
```
Comment (Schplee) [inline games/lego-city-undercover-the-chase-begins/game.dat]: Release dates shouldn't be included in game descriptions.

File: games/lego-city-undercover-the-chase-begins/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "00040000000AD500"
+region = "usa"
+release_date = "2013-03-18"
```
Comment (Schplee) [inline games/lego-city-undercover-the-chase-begins/game.dat]: The release date is incorrect. The correct release date is: `NA: 21 April 2013`

File: games/lego-city-undercover-the-chase-begins/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+[[testcases]]
+title = "00040000000AD500"
+compatibility = "3"
+date = "2017-11-08"
+version = "HEAD-ddee442"
+cpu = "AMD FX 5600 Quad"
+gpu = "AMD R9 270"
+os = "Windows 10 10.0.15063"
```
Comment (Schplee) [inline games/lego-city-undercover-the-chase-begins/game.dat]: Should be `Windows 10 15063`

File: games/lego-city-undercover-the-chase-begins/game.dat (hunk 1 above)
Comment (EzekielRage) [reply games/lego-city-undercover-the-chase-begins/game.dat]: fixed^^

File: games/lego-city-undercover-the-chase-begins/game.dat (hunk 2 above)
Comment (EzekielRage) [reply games/lego-city-undercover-the-chase-begins/game.dat]: fixed^^

File: games/lego-city-undercover-the-chase-begins/game.dat (hunk 3 above)
Comment (EzekielRage) [reply games/lego-city-undercover-the-chase-begins/game.dat]: fixed^^

**Comment by Schplee** [comment]
LGTM

**Comment by Schplee** [comment]
LGTM

**Comment by Leo626** [comment]
LGTM but I feel the issues described in the wiki should be reported and added though this can be done later.

File: games/lego-city-undercover-the-chase-begins/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+title = "00040000000AD500"
+region = "usa"
+release_date = "2013-04-21"
+
+[[testcases]]
+title = "00040000000AD500"
+compatibility = "3"
+date = "2017-11-08"
+version = "HEAD-ddee442"
+cpu = "AMD FX 5600 Quad"
```
Comment (TheKoopaKingdom) [inline games/lego-city-undercover-the-chase-begins/game.dat]: Nit: for consistency, this might be better as `Quad Core`, or without the `Quad`.

**Pull request merged**